### ISSUE TITLE
[Enhancement] Catalan flag 🏴󠁥󠁳󠁣󠁴󠁿

### ISSUE THREAD

**Issue by Alzavio** [twitter/twemoji#574]
**Is your feature request related to a problem? Please describe.**

While not encouraged, it's become somewhat standard to have flags represent languages. Catalan being one of the most widespreadly spoken languages in Europe, meaning it's supported in a lot of applications. Though it doesn't have an emoji flag to represent it, which presents some annoying workarounds each time it needs to be added.

**Describe the solution you'd like**

I'm requesting the Catalan flag to be added, similar to how Wales, and Scotland were. There was an old thread about this before Wales was added with someones attempt, but I don't know how to reopen it.

https://github.com/twitter/twemoji/issues/282

**Describe alternatives you've considered**
Creating and using a modified svg of the Catalan flag, however doing that often just for Catalan is annoying

**Comment by snessim** [comment]
Per the Twemoji readme: "We do not support custom emoji." 
**This emoji should instead be [proposed here](https://unicode.org/emoji/proposals.html).**
For a full list of all emoji (including flags) found in the current version of Unicode, [see the Unicode website](https://unicode.org/emoji/charts/full-emoji-list.html#1fae0).

**Comment by Alzavio** [comment]
@catgirl-doll I may just be misunderstanding but the flag of England (🏴󠁧󠁢󠁥󠁮󠁧󠁿) is [U+1F3F4 U+E0067 U+E0062 U+E0065 U+E006E U+E0067 U+E007F](https://unicode.org/emoji/charts/full-emoji-list.html#1f3f4_e0067_e0062_e0065_e006e_e0067_e007f)

Isn't the [Catalan emoji](https://emojipedia.org/flag-for-catalonia-esct/) (🏴󠁥󠁳󠁣󠁴󠁿) just the same sort of thing with the following? U+1F3F4 U+E0065 U+E0073 U+E0063 U+E0074 U+E007F

**Comment by kkiris** [comment]
The flags for England, Scotland, & Wales are the only subnational flags that are currently recommended for general interchange (RGI) by Unicode. However, flags for other subdivisions **_are_** considered valid by Unicode. These use tag sequences (just like the flags for England, Scotland, & Wales do). See [Unicode Technical Standard #51: Annex C](https://unicode.org/reports/tr51/#flag-emoji-tag-sequences) for more information. This annex uses the flags for California, Ontario, Canton Zürich, & Normandy as examples, but the same would apply to the flag for Catalonia, as tag sequences are valid for any subdivision, provided it has an [ISO 3166-2 code](https://en.wikipedia.org/wiki/ISO_3166-2).

@Alzavio is correct that `U+1F3F4 U+E0065 U+E0073 U+E0063 U+E0074 U+E007F` is the codepoint sequence for the flag for Catalonia (🏴󠁥󠁳󠁣󠁴󠁿), as that tag sequence corresponds to Catalonia's [3166-2 code](https://www.iso.org/obp/ui/#iso:code:3166:ES).

**tl;dr**: This wouldn't be a custom emoji. This is a valid use of the supported tag sequence mechanism for subnational flag emojis; though, this emoji isn't currently supported by most emoji libraries.
